deploy/deploy.sh

```diff
@@ -35,14 +35,16 @@ defaultDriverImage="quay.io/openshift/origin-csi-driver-shared-resource:4.9.0"
 driverImage=${DRIVER_IMAGE:-${defaultDriverImage}}
 
 cp -r "${BASE_DIR}"/* "${DEPLOY_DIR}"
+# TODO: Remove below line before merging after openshift/api#979 merges
```

Comment:
how does the api merging help with this?

per my comment in the Makefile, this can go away once api#979 is vendored into openshift/openshift-apiserver

Reply:
agreed

Reply:
do you want to update this comment like you did the other one ?